client/lcd/swagger-ui/swagger.yaml

```diff
@@ -217,7 +217,7 @@ paths:
         - in: query
           name: tag
           type: string
-          description: "transaction tags such as 'action=submit-proposal' and 'proposer=cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc' which results in the following endpoint: 'GET /txs?action=submit-proposal&proposer=cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc'"
+          description: "transaction tags such as 'action=submit-proposal&proposer=cosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc' which results in the following endpoint: 'GET /txs?tag=action%3Dsubmit-proposal%26proposer%3Dcosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc'"
```

Comment:
As I wrote down, I'm not sure there is a use-case for this, there is a benefit in adding a tag that contains other GET variables urlencoded?

Reply:
As I reply upstairs. :)

Reply:
> 'GET /txs?tag=action%3Dsubmit-proposal%26proposer%3Dcosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc'"

I don't really like this implementation

Reply:
> > 'GET /txs?tag=action%3Dsubmit-proposal%26proposer%3Dcosmos1g9ahr6xhht5rmqven628nklxluzyv8z9jqjcmc'"
> 
> I don't really like this implementation

It just was generated by swagger, not for human. Please refer to https://github.com/cosmos/cosmos-sdk/issues/3990#issuecomment-478024691